CMSgov/cms-carts-seds
# [Mdct 197] 2022 Forms & PDF

# Description

[Ticket](https://qmacbis.atlassian.net/browse/MDCT-197)

Links to the new pdf.
Forms can be generated for states for 2022.
Includes a fix for the 2022 template for jsonpath updates, and for how radio buttons display children (now they do.)
Seeds the 2022 forms into each env on deploy.

## How to test

Check out the deployed branch here: https://d3dbtfz10nfh9d.cloudfront.net/

It already has the 2022 forms generated, so feel free to click around. 
If you're really feeling generous, check out the red-lined PDF attached to the ticket and double check that differences were adjusted.

Happy to walk through this branch with anyone who cares.

## Dependencies

None

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
